script/install.sh

```diff
@@ -27,7 +28,10 @@ echo "1.create directory"
 # If install Path equal to "/" or related path is "/" or is empty, will cause directory "/bin" be overwrite or file adding,
 # so we should check its value. Here use command `realpath` to get the related path, and it will skip if your shell env
 # without command `realpath`.
-if [ ! -d $installPath ];then
+if [ ${baseDir} = $installPath ]; then
+  echo "Parameter installPath can not be same as the current path (pwd)."
```

Comment:
echo "Fatal: The installPath can not be same as the current path: ${installPath}"  may be better, WDYT.

Reply:
@EricGao888 @caishunfeng PTAL

Reply:
good point